CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Support for keybindings with dead keys
 - `Back`/`Forward` mouse buttons support in bindings
 - Copy global IPC options (`-w -1`) for new windows
+- Environment variable `EGL_OVER_GLX` for setting display api priority
```

Comment:
We should call this variable like `PREFER_EGL`, and make it works reliably across the platforms.

Reply:
Do you mean that the flag should affect macOs and Windows systems as well?

Reply:
Yes, it should work on all platforms.